lca-cli/postpivot/postpivot.go

```diff
@@ -58,6 +59,7 @@ const (
 	sshKeyEarlyAccessFile = "/home/core/.ssh/authorized_keys.d/ib-early-access"
 	userCore              = "core"
 	sshMachineConfig      = "99-%s-ssh"
+	pullSecretFileName    = "pullsecret.json"
```

Comment:
What is the significance of this file? Why are we creating it?

Reply:
same as with machine config, adding it to manifests folder and will apply everything from there. 
Will add comment to the function

Reply:
Why do we need to create the file? why not create the secret from code?
Also note that @leo8a is removing the secret from the seed [here](https://github.com/openshift-kni/lifecycle-agent/pull/256)

Reply:
We can do it but i think we will need first of all to create docker config on filesystem, after Leo will remove old ps (assumption) and in this case this logic will be the same as with ssh key, we handle everything the same way

Reply:
I don't see how creating a secret file to be applied to k8s relates to the docker config file (just the secret value decoded in json format) that we should place on the filesystem

Reply:
@tsorya it's worth adding this :point_up: as a comment in the code (above line 84)